automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -19,8 +19,8 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
 
   "Orchestration" - {
     "should grant and remove google role access" in {
-      // google roles may take a little while to take effect
-      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(3, Minutes)), interval = scaled(Span(1, Second)))
+      // google roles can take a while to take effect
+      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Minutes)), interval = scaled(Span(10, Seconds)))
```

Comment:
Is there a reason for bumping the interval to 10 seconds?

Reply:
Not specifically, just looking to avoid hammering Google.  I can switch it back.

Reply:
Sounds like a good idea

Reply:
Meh, It's not too important to me either way. You could also choose a 3 or 5 second interval if you want.